dao.js

```diff
@@ -0,0 +1,51 @@
+var AWS = require("aws-sdk");
+
+AWS.config.update({
+  region: "us-west-1",
+  endpoint: "http://localhost:8000"
+});
+
+var dynamo = function(req, res) {
+    this.tableName = "Accounts";
+    this.docClient = new AWS.DynamoDB.DocumentClient();
+}
+
+//var tableName = "Accounts";
+//var docClient = new AWS.DynamoDB.DocumentClient();
+
+dynamo.prototype.putUser = function(username, password) {
+    var params = {
+        TableName: this.tableName,
+        Item : {
+            "Username" : username,
+            "Password" : password
```

Comment:
```suggestion
        tableName: this.tableName,
        item : {
            "username" : username,
            "password" : password
```
Our style guide says: "parameters and local variables are all lowerCamelCase." Is this something specifically needed for setting up the database? I think these are like parameters, but now I'm a bit confused if the upper case letter is for something specific

Reply:
Sounds like the upper camel case is needed for the database.